sonic_platform_base/sonic_ssd/ssd_generic.py

```diff
@@ -23,6 +23,8 @@
 # Set Vendor Specific IDs
 INNODISK_HEALTH_ID = 169
 INNODISK_TEMPERATURE_ID = 194
+SWISSBIT_HEALTH_ID = 248
+SWISSBIT_TEMPERATURE_ID = 194
```

Comment:
@Junchao-Mellanox where is temp id used?

Reply:
Fixed